src/frontend/teachers-side/src/app/authentication/components/login/login.component.scss

```
@@ -0,0 +1,25 @@
+form { margin-right:50%;
+  margin-top:-35%;
+  padding-left:10%;}
```

Please format your code properly so it's readable
```suggestion
form { 
  margin-right: 50%;
  margin-top: -35%;
  padding-left: 10%;
}
```